### ISSUE TITLE
Sporadic behaviour with 'ghi list' while 'ghi list -v' is consistent

### ISSUE THREAD

**Issue by jasonhargrove** [stephencelis/ghi#176]
Occasionally, on some terminal windows, 'ghi list' returns a list of issues.
Most of the time it doesn't return the list, and simply says:
```
# me/myRepo open issues
```
And then just the prompt.

In all cases, 'ghi list -v' returns the verbose list. I can show an item, open items, etc. It's just the list function that is hit and miss.

**Comment by haakenlid** [comment]
It seems as the list output doesn't show if the terminal window is too narrow. In my case the list will only show up in a terminal at least 82 characters wide.

**Comment by PeterWhittaker** [comment]
While leaving the terminal width the same, can you try adjusting the value of the COLUMNS environment variable and see if that makes any difference, e.g.,

$ COLUMNS=80 ghi list
$ COLUMNS=20 ghi list
$ COLUMNS=120 ghi list

**Comment by haakenlid** [comment]
The problems seems to be when I use many labels, or labels with long names on an issue. 
When the total space taken by all the labels for one issue is wider than the available columns, the bug appears.

**Comment by haakenlid** [comment]
using `PAGER=cat` resulted in a stack trace that was not shown with the default pager. 

    /usr/local/bin/ghi:1929:in `truncate': undefined method `strip' for nil:NilClass (NoMethodError)
            from /usr/local/bin/ghi:2033:in `block in format_issues'
            from /usr/local/bin/ghi:2021:in `map'
            from /usr/local/bin/ghi:2021:in `format_issues'
            from /usr/local/bin/ghi:3731:in `block in execute'
            from /usr/local/bin/ghi:1905:in `block in page'
            from /usr/local/bin/ghi:1904:in `loop'
            from /usr/local/bin/ghi:1904:in `page'
            from /usr/local/bin/ghi:3714:in `execute'
            from /usr/local/bin/ghi:2745:in `execute'
            from /usr/local/bin/ghi:91:in `execute'
            from /usr/local/bin/ghi:4133:in `<main>'

**Comment by PeterWhittaker** [comment]
What was the complete command line? I just tried 

  PAGER=cat ghi list
  PAGER=cat COLUMNS=20 ghi list
  PAGER=cat COLUMNS=120 ghi list
  PAGER=cat ghi list -v
  PAGER=cat COLUMNS=20 ghi list -v
  PAGER=cat COLUMNS=120 ghi list -v

all of which succeeded. I have ghi version 0.9.3, running under ruby 2.1.5p273 (2014-11-13 revision 48405) [x86_64-darwin13.0], under bash 4.3.33(1)-release (x86_64-apple-darwin14.1.0). No idea of the version of cat, cannot tell, it's whatever OS X ships.

**Comment by haakenlid** [comment]
First I open an issue with a label that is too wide for the terminal:
```bash
$ ghi open -m 'test bug #176' -L 'very very very long label that cannot fit inside the terminal window since it soooooooooooooooooooooooooo LONG'
```
Then I make sure the terminal window is too narrow for the label text, and run a regular ghi list, PAGER and COLUMNS doesn't have any effect on the bug, but only `cat` shows the stacktrace.
```bash
$ tput cols
80 

$ PAGER=cat ghi
# user/repo open issues
/usr/local/bin/ghi:1929:in `truncate': undefined method `strip' for nil:NilClass (NoMethodError)
        from /usr/local/bin/ghi:2033:in `block in format_issues'
        from /usr/local/bin/ghi:2021:in `map'
        from /usr/local/bin/ghi:2021:in `format_issues'
        from /usr/local/bin/ghi:3731:in `block in execute'
        from /usr/local/bin/ghi:1905:in `block in page'
        from /usr/local/bin/ghi:1904:in `loop'
        from /usr/local/bin/ghi:1904:in `page'
        from /usr/local/bin/ghi:3714:in `execute'
        from /usr/local/bin/ghi:2745:in `execute'
        from /usr/local/bin/ghi:91:in `execute'
        from /usr/local/bin/ghi:4133:in `<main>'
```

**Comment by AlexChesters** [comment]
Hi all,

This merged PR today 257375e12af74b9ac5227c05f413827ee2a15f8c should fix this issue. Please reopen if it doesn't.

Thanks a lot!
